books&lectures/(인프런)스프링 핵심 원리 고급편/src/main/java/hello/springadvanced/v0/OrderRepositoryV0.java

```diff
@@ -24,4 +24,30 @@ private void sleep(int millis) {
             e.printStackTrace();
         }
     }
+
+    public void save2(String s) {
+        if (s == "테스트") {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**: An equality operator (== or !=) is used to compare strings, which matches strings based on address.

**Fix**: Do content comparison using the equals() method to compare the values of the strings.